EIPS/eip-7779.md

```diff
@@ -0,0 +1,91 @@
+---
+eip: 7779
```

Comment:
```suggestion
eip: 7783
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.